openff/toolkit/typing/engines/smirnoff/parameters.py

```diff
@@ -1735,17 +1729,6 @@ class ParameterType(_ParameterAttributeHandler):
     id = ParameterAttribute(default=None)
     parent_id = ParameterAttribute(default=None)
 
-    @smirks.converter
-    def smirks(self, attr, smirks):
-        # Validate the SMIRKS string to ensure it matches the expected
-        # parameter type, raising an exception if it is invalid or doesn't
-        # tag a valid set of atoms.
-
-        # TODO: Add check to make sure we can't make tree non-hierarchical
-        #       This would require parameter type knows which ParameterList it belongs to
-        ChemicalEnvironment.validate_smirks(smirks, validate_valence_type=True)
-        return smirks
-
```

Comment:
Note to self: This should be fine to remove, since all ParameterType classes would have been calling this with the base ChemicalEnvironment class, which had a _VALENCE_TYPE of None (so this check wasn't doing anything)